app/models/work_version.rb

```diff
@@ -59,6 +59,9 @@ class WorkVersion < ApplicationRecord
     after_transition on: :submit_for_review, do: WorkObserver.method(:after_submit_for_review)
     after_transition on: :deposit_complete, do: WorkObserver.method(:after_deposit_complete)
     after_transition on: :deposit_complete, do: CollectionObserver.method(:item_deposited)
+    after_transition on: :decommission do |work_version, _transition|
+      WorksMailer.with(work: work_version.work).decommission_email.deliver_later
```

Comment:
Would you mind putting this in the WorkObserver.after_decommission method to follow the pattern here?